docs/admin-guides/managing-optimizers.md

```diff
@@ -30,6 +30,19 @@ containers:
       export.JAVA_HOME: "/opt/java"   # JDK environment
 ```
 
+### KubernetesOptimizerContainer container
+
+Kubernetes Optimizer Container is a way to start Optimizer On K8s with standalone Optimizer.
+
+```yaml
+containers:
+  - name: KubernetesContainer
+    container-impl: com.netease.arctic.server.manager.KubernetesOptimizerContainer
+    properties:
+      kube-config-path: ～/.kube/config
+      image: arctic163/amoro:0.6
+```
```

Comment:
We should support assign namespace. And user could assign namespace in `$AMS_HOME/conf/config.yaml` or `ResourceGroup`'s properties. 

The resource group has high priority.